TipBuilder/Lua/Autorun/tipBuilder.lua

```diff
@@ -0,0 +1,51 @@
+-- by Television (most functions were made by Evil Factory, modified by me)
+-- sends a message to the server every 500 seconds
+
+if CLIENT then return end
+
+-- CONFIG
+local messageIsPopUp = false -- if the tips are popups or not
+local time = 0 -- don't change
+local tipDelay = 500 -- change 500 to how much longer you want the next tip to be (in seconds)
+local TipText = "Pro Tip: " -- make this string blank if you don't want something before the tip
+
+local Tips = {
+    "Join our discord!",
+    "hi"
+}
+
+SendChatMessage = function (client, text, color)
```

Comment:
This is polluting the global space, either make the function local or add it to some namespace table